modules/op-using-privileged-builds.adoc

```diff
@@ -0,0 +1,64 @@
+==== Running pods without privileged security context (default)
+
+The default configuration of OpenShift Pipelines 1.3.x and 1.4.x  does not allow you to run pods  with privileged security context, if the pods result from pipeline run or task run.
+For such pods, the default service account is `pipeline`, and the Security Context Constraint (SCC) associated with the `pipelines` service account is `pipelines-scc`. The `pipelines-scc` service account is similar to `anyuid`, but with a minor difference as defined in the YAML file for SCC of pipelines:
+
+[source, YAML]
+----
+fsGroup:
+  type: MustRunAs
+----
+In addition, the `Buildah` cluster task shipped as part of the OpenShift Pipelines uses `vfs` as the default storage driver.
+
+==== Running pods with privileged security context
+
+To run a pod (resulting from pipeline run or task run) with `privileged` security context, do the following modifications:
+
+* Configure the associated user account or service account to have an explicit SCC. You can perform the configuration using any of the following methods:
+** Use the OpenShift command below:
++
+[source, terminal]
+----
+$ oc adm policy add-scc-to-user <scc-name-here> -z <service-account-name>
+----
+** Manually modifying the YAML for `RoleBinding` and `Role` or `ClusterRole`:
```

Comment:
This is confusing to me... 

1. I would expect these two methods to be equal. `oc` command shows `<service-account-name>` while yaml shows `pipeline`
2. `Manually modifying the YAML`... are we guiding the customer to update existing thing or to create a new one? Rolebinding named `foo` suggests that we are creating something new but cluster role `pipelines-scc-clusterrole` suggest that we are updating the one shipped with Pipelines
3. metadata in cluster role is empty, I would add at least name field (has to match the one in role binding)